driver/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ add_executable(MIOpenDriver
     dm_fusion.cpp
     dm_gemm.cpp
     dm_groupnorm.cpp
+    dm_multimarginloss.cpp
```

Comment:
You should sort file name in lexical order.

Reply:
wow ok very detailed review. I forget that M is after L in alphabet